roles/slurm/files/epilog.d/50-exclusive-parts

```diff
@@ -5,7 +5,7 @@ numcpus_sys=$(( $(grep -c ^processor /proc/cpuinfo) * $(scontrol show job "$SLUR
 numcpus_job=$(scontrol show job "$SLURM_JOBID" | grep -Eio "TRES=cpu=[0-9]+" | cut -d= -f3)
 if [ "$numcpus_sys" == "$numcpus_job" ] ; then
     logger -s -t slurm-epilog 'cleaning up after exclusive job'
-    run-parts -v --reverse /etc/slurm/epilog-exclusive.d
+    run-parts /etc/slurm/epilog-exclusive.d
```

Comment:
@dholt I had the `--reverse` here so that the priority ordering will match between linux's `run-parts` and slurm's equivalent. Since you removed it in this PR, the order of execution is now different in different folders.

Reply:
I don't remember the reason but I believe we had to remove this to fix a bug

Reply:
@lukeyeager I just remembered the reason, `run-parts` on CentOS doesn't accept these flags.

Reply:
Ah. Darn. Ok.